### PR TITLE
Starter site based on Hugo

### PR DESCRIPTION
**Pull request by pdmack**
/assign @ewilderj 

Edd, really just a rough-cut, Hugo-friendly site with a reasonable landing page and logos. I don't know that much about Hugo. More or less a sandbox by which we could maybe start adding MD content. 

The theme is called "[kube](https://themes.gohugo.io/theme/kube/)" and the developer seemed to be channeling fonts and colors from Kubernetes. Not too sure about any other inferred connections. 

This first PR carries in a lot of vestiges of the sample theme and I think that the layout has more in there than what is needed. But it renders fine on my Mac...just need to be careful about what gets thrown away.

screen shot 2018-04-18 at 9 49 00 am

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # website
 Kubeflow's public website
+
+## Development
+
+### Getting started
+macOS...sorry :-)
```
Comment (jlewi) [inline README.md]: Hilarious.

**Comment by jlewi** [comment]
/assign @texasmichelle 
/assign @nickchase 
/unassign jllewi

I'm going to let others take the lead on reviewing this. Happy to rubber stamp though.

**Comment by ewilderj** [comment]
Did you include the code for the front page screenshot? I can just see default Kube theme content.

**Comment by willingc** [comment]
I had the same issue as @ewilderj with the photos. The menus seemed fine. I think the git submodule didn't update properly.

**Comment by pdmack** [comment]
Uhhhh.
Not sure what's going on.
Clean pull, render, and it's crap.
Not sure what's going on.
Almost seems like there was some caching in play.
Let me get back to you.
This is what I was wondering about...how Hugo was scanning the dirs for serving content.
BTW, no submodules.

**Comment by pdmack** [comment]
OK, weird. The `git add` actually left out dirs from the commit. Stay tuned for Round #2.

**Comment by pdmack** [comment]
@ewilderj @willingc PTAL
Apologies for the goat rodeo. Indeed, I got tripped by that old git chestnut: the "phantom" submodule.
I'm still not sure I understand how Hugo is interacting with the theme and content at the file level overall but probably time for me to slowly step away from the keyboard and let someone smart like @willingc take over.

**Comment by willingc** [comment]
@pdmack Thanks for the update. It works 👍 

Full disclosure: It took me weeks to fully understand the things you mention about interaction of theme and content. Changing themes would throw me for a loop. I also couldn't see where stuff was coming from and how to override. 

Recommend merging this PR and we can clean up the stock content in a future PR. Thanks @pdmack :sunny:

**Comment by jlewi** [comment]
I'll manually merge this PR. It looks like tide wasn't enabled for this repo (#4). We should fix that so merging happens automatically.

**Pull request merged**